improver_tests/utilities/test_TemporalInterpolation.py

```diff
@@ -113,41 +125,42 @@ def setUp(self):
         self.times = []
         for i in range(4, 9):
             self.times.append(datetime.datetime(2017, 11, 1, i))
-        self.expected = [('time', [time for time in self.times])]
+        self.expected = [("time", [time for time in self.times])]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of a comprehension](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5385606)